File: Sample.ElectronicCommerce.BrokerMail/Entities/EF/Mapping/MailBrokerEntity.cs
using System;

namespace Sample.ElectronicCommerce.BrokerMail.Entities.EF.Mapping
{
    public class MailBrokerEntity
    {        
        public long Id { get; set; }

		public DateTime? DtCreation { get; set; }

		public DateTime? DtLastUpdate { get; set; }		

		public string Server { get; set; }

		public string Name { get; set; }

		public string Password { get; set; }

		public string UserName { get; set; }

		public string Code { get; set; }

		public int? Port { get; set; }

		public bool IsEnabledSsl { get; set; }

		public bool IsActive { get; set; }
	}
}

File: Sample.ElectronicCommerce.Web/ClientSide/src/app/shared/util/Entities/MailBrokerEntity.ts
export class MailBrokerEntity {
    public id: number = 0;
    public dtCreation: Date = null;
    public dtLastUpdate: Date = null;    
    public name: string = '';
    public server: string = '';
    public userName: string = '';
    public password: string = '';    
    public code: string = '';
    public port: number = null;
    public isEnabledSsl: boolean = true;
    public isActive: boolean = true;
}